libs/template/templates/lakeflow-pipelines/databricks_template_schema.json

```diff
@@ -1,57 +1,96 @@
 {
-    "welcome_message": "\nWelcome to the template for Lakeflow Declarative Pipelines!",
+    "//": "This template is based on an upcoming version of the default-python template, but comes with a SQL variation + certain defaults that are specific to Lakeflow Pipelines.",
+    "welcome_message": "Welcome to the template for Lakeflow Declarative Pipelines!\n\nPlease answer the below to tailor your project to your preferences.\nYou can always change your mind and change your configuration in the databricks.yml file later.\n\nNote that {{workspace_host}} is used for initialization\n(see https://docs.databricks.com/dev-tools/cli/profiles.html for how to change your profile).",
     "properties": {
+        "lakeflow_only": {
```

Comment:
What is the intent? Use from the UI only?

Reply:
Yeah, exactly, this is just for retrofitting the `default-python` template to be suitable as a replacement for `lakeflow-pipelines`.